application/src/main/java/org/togetherjava/tjbot/commands/free/ChannelMonitor.java

```diff
@@ -0,0 +1,245 @@
+        List<ChannelStatus> statusFor = guildMonitoredChannelsList(guild);
+
+        // update name so that current channel name is used
+        statusFor.forEach(channelStatus -> channelStatus.updateChannelName(guild));
+
+        // dynamically separate channels by channel categories
+        StringBuilder sb = new StringBuilder();
+        String categoryName = "";
+        for (ChannelStatus status : statusFor) {
+            Category category = guild.getGuildChannelById(status.getChannelId()).getParent();
+            if (category != null && !category.getName().equals(categoryName)) {
+                categoryName = category.getName();
+                // append the category name on a new line with markup for underlining
+                // fixme possible bug when not all channels are part of categories, may mistakenly
+                // include uncategoried channels inside previous category. will an uncategoried
+                // channel return an empty string or null? javadocs dont say.
+                sb.append("\n__").append(categoryName).append("__\n");
+            }
+            sb.append(status.toDiscord()).append("\n");
+        }
+
+        return sb.toString();
+    }
+
+    /**
+     * This method checks all channels in a guild that is currently being monitored and are
+     * currently busy and determines if the last time it was updated is more than an hour ago. If so
+     * it changes the channel's status to free.
+     * <p>
+     * This method is run automatically during startup and should be run on a 15minute schedule. The
+     * scheduled execution is not currently implemented
+     * 
+     * @param guild the guild for which to test the channel statuses of.
+     */
+    public void updateStatusFor(@NotNull Guild guild) {
+        List<ChannelStatus> statusFor = guildMonitoredChannelsList(guild);
+
+        statusFor.stream()
+            .parallel()
+            .filter(ChannelStatus::isBusy)
+            .map(ChannelStatus::getChannelId)
+            .map(guild::getTextChannelById)
+            .filter(this::isChannelInactive)
+            .map(TextChannel::getIdLong)
+            .forEach(this::setChannelFree);
+    }
+
+    /**
+     * This method returns the {@link TextChannel} that has been configured as the output of the
+     * status messages about busy/free for the specified guild.
+     *
+     * @param guild the {@link Guild} for which to retrieve the TextChannel for.
+     * @return the TextChannel where status messages are output in the specified guild.
+     */
+    public TextChannel getStatusChannelFor(@NotNull final Guild guild) {
+        // todo add error checking for invalid keys ??
+        return guild.getTextChannelById(postStatusInChannel.get(guild.getIdLong()));
+    }
+
+    /**
+     * The toString method for this class, it prints out a list of the currently monitored channels
+     * and the channels the status are printed in. This is called on boot by as a debug level logger
```

Comment:
`"This is called on boot by as a debug level logger"` should not be here. if you think its interesting information, maybe add it as a comment within the method, like `// NOTE This is called on ...`.